src/layouts/Root/components/Header.tsx

```diff
@@ -80,14 +80,14 @@ const Logo: React.FC = () => (
           css={css`
             width: ${p => p.theme.iconSizes.lg};
             height: ${p => p.theme.iconSizes.lg};
-            color: ${PALETTE.green[400]};
+            color: ${p => getColorV8('green', 400, p.theme)};
           `}
         />
         <GardenWordmark
           css={css`
             margin-left: ${p => p.theme.space.xs};
             height: ${p => p.theme.iconSizes.lg};
-            color: ${PALETTE.kale[700]};
+            color: ${p => getColorV8('kale', 700, p.theme)};
```

Comment:
```suggestion
            color: ${p => getColor({ theme: p.theme, hue: 'kale', shade: 900 })};
```

Reply:
ditto on leaving website product colors until design has a chance to visit the comps.